spec/system/admin/settings_spec.rb

```diff
@@ -340,4 +341,30 @@
       Setting["feature.user.skip_verification"] = nil
     end
   end
+
+  describe "SDG configuration tab" do
+    scenario "is enabled when the sdg feature is enabled" do
+      Setting["feature.sdg"] = true
+      admin = create(:administrator).user
+      login_as(admin)
+
+      visit admin_settings_path
+      find("#sdg-tab").click
+
+      expect(page).to have_content("SDG configuration")
```

Comment:
This test passes even without clicking the "SDG configuration" link, since the text is already present on the page (it's the same text as in the link.

How about `expect(page).to have_css "h2", exact_text: "SDG configuration"`